app/javascript/components/mentoring/automation/RepresentationList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Pagination } from '../../common/Pagination'
+import { FetchingBoundary } from '../../FetchingBoundary'
+import { APIResponse } from '../queue/useMentoringQueue'
+import { QueryStatus } from 'react-query'
+import { AutomationListElement } from './AutomationListElement'
+
+const DEFAULT_ERROR = new Error('Unable to fetch queue')
+
+type Props = {
+  resolvedData: APIResponse | undefined
+  latestData: APIResponse | undefined
+  page: number
+  setPage: (page: number) => void
+}
+
+export const RepresentationList = ({
+  status,
+  error,
+  ...props
+}: { status: QueryStatus; error: unknown } & Props): JSX.Element => {
+  return (
+    <FetchingBoundary
+      status={status}
+      error={error}
+      defaultError={DEFAULT_ERROR}
+    >
+      <Component {...props} />
+    </FetchingBoundary>
+  )
+}
+
+const Component = ({ resolvedData, latestData, page, setPage }: Props) => {
+  return (
+    <>
+      {resolvedData && resolvedData.results && (
+        <React.Fragment>
```

Comment:
Let's be consistent with either `<>` or `<React.Fragment>`. I prefer the first personally but I'm not fussy.

Reply:
I also prefer the first.